app/helpers/SessionAnswersHelper.scala

```diff
@@ -473,4 +474,19 @@ class SessionAnswersHelper @Inject()(uploadJourneyRepository: UploadJourneyRepos
     userRequest.answers.getAnswer[String](SessionKeys.doYouWantToAppealBothPenalties).contains("no") &&
       userRequest.answers.getAnswer[LocalDate](SessionKeys.dateCommunicationSent).exists(_.isBefore(dateTimeNow.minusDays(appConfig.daysRequiredForLateAppeal)))
   }
+
+  def viewAppealRows()(implicit userRequest: UserRequest[_], messages: Messages): Seq[CheckYourAnswersRow] = {
+     Seq(CheckYourAnswersRow(
```

Comment:
Having a 'CheckYourAnswersRow' seems a bit strange, maybe rename the model to be 'QuestionAnswerRow'?